### ISSUE TITLE
console - UI shows user_name lower while the true user_name string could be upper or lower

### ISSUE THREAD

**Issue by MaelREBOUX** [georchestra/georchestra#2371]
**Describe the bug**
We met a big problem to follow user connexions.

When creating an account, new user may change the proposed string login, wich is in lower string.
If the user change it and put upper case caracter : the LDAP wil store this string with upper OR mixed caracters.

The problem is that the UI interface of the console shows an only lower case tranformed string.
**The code that produces the /ldapadmin/private/users/ json lead us wrong with the user_name users.**

Use case : I recently considered that a user ''rburgaleta" made no connection to georchestra because I found no trace in the ogc_statistics tables. SO I delete some permissions to him.
In reality his user_name in LDAP and the system is "RBurgaleta".

**geOrchestra version or branch**

17.12  18.06

**Expected behavior**
**The console MUST show real user_name.**

**Comment by MaelREBOUX** [comment]
Since 18.06 the url of the jso users list is  /console/private/users

**Comment by fvanderbiest** [comment]
FYI, 18.12 will bring the ability to forbid the user to modify its login, see https://github.com/georchestra/datadir/blob/18.12/console/console.properties#L29-L31 and #2162

**Comment by MaelREBOUX** [comment]
**Could you please QUICKLY fix the console as it show the REAL login of the users ?**

We have deleted dozens of accounts because we based our analyse on the user_name provided by the console. But the true login (case sensible) is stored in the OGC log tables.

**Comment by MaelREBOUX** [comment]
No : the problem is on the **user ID**. But the attribute is named **user_name** in the JSON. Not my fault.

**Comment by fvanderbiest** [comment]
The issue seems server-side since the /console/private/users webservice produces lowercased uid 

Eg for user `FVanderbiest`:
```json
{
uid: "fvanderbiest",
givenName: "François",
sn: "Van Der Biest",
org: "Camptocamp",
orgId: "C2C",
mail: "f...@camptocamp.com",
pending: true,
privacyPolicyAgreementDate: null
}
```

**Comment by arnaud-morvan** [comment]
If I correctly understand, username is always stored lowercase in LDAP.

But user can login with any case. For example I can login in georchestra/docker with TestAdmin / testadmin.

In such case username is returned as "TestAdmin" by CAS, so it is also logged in ogcstatistics as "TestAdmin".

So we can propose to convert username to lower case before adding records to ogcstatistics: #3795

Note that it might be possible to directly update existing table content if needed: 
```
UPDATE ogcstatistics.ogc_services_log
SET user_name = lower(user_name)
WHERE user_name != 'anonymousUser';
```

Regards

**Comment by pmauduit** [comment]
we will probably need to fix it into the 22.0.x branch as well

**Comment by pmauduit** [comment]
> we will probably need to fix it into the 22.0.x branch as well

https://github.com/georchestra/georchestra/pull/3798

**Comment by MaelREBOUX** [comment]
Love it

**Comment by MaelREBOUX** [comment]
So : the login is case insensitive. What a discover for me.

**Comment by arnaud-morvan** [comment]
I think we can close this ticket as it has been merged in branches 22.0.x and master.
Will be available in next release.
Feel free to reopen if needed.

**Comment by fvanderbiest** [comment]
> If I correctly understand, username is always stored lowercase in LDAP.

Should we prevent users from creating logins with uppercased chars then ?
